### PR TITLE
Added publishing.gradle files with publications metadata.

### PR DESCRIPTION
**Pull request by ThomasRitense**

### CODE REVIEW

File: valtimo-dependencies/gradle/publishing.gradle [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Valtimo dependencies module'
+                    description = 'Valtimo dependencies module contains all modules that are part of Valtimo'
```
Comment (marijnritense) [inline valtimo-dependencies/gradle/publishing.gradle:23]: Maybe change this to something like:

Valtimo dependencies module containing all the mandatory modules for Valtimo. Also contains version constraints.

File: valtimo-gzac-dependencies/gradle/publishing.gradle [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Valtimo GZAC dependencies module'
+                    description = 'Valtimo GZAC dependencies module contains all modules that are part of the Valtimo GZAC edition'
```
Comment (marijnritense) [inline valtimo-gzac-dependencies/gradle/publishing.gradle:23]: Maybe change this to something like:

Valtimo dependencies module containing all the mandatory modules for Valtimo (GZAC edition). Also contains version constraints.

File: valtimo-gzac-dependencies/gradle/publishing.gradle [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Valtimo GZAC dependencies module'
```
Comment (marijnritense) [inline valtimo-gzac-dependencies/gradle/publishing.gradle:22]: Not too sure on the name. While it fits the naming scheme of all of the other modules we have, I don't think using "module" in the name really adds something in this case.

File: valtimo-dependencies/gradle/publishing.gradle [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Valtimo dependencies module'
```
Comment (marijnritense) [inline valtimo-dependencies/gradle/publishing.gradle:22]: Not too sure on the name. While it fits the naming scheme of all of the other modules we have, I don't think using "module" in the name really adds something in this case.

**Review by Klaas-Ritense — APPROVED** [review]

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**